docs/user/troubleshooting.md

```diff
@@ -145,3 +145,105 @@ Images:
 ```
 
 [kubernetes-debug]: https://kubernetes.io/docs/tasks/debug-application-cluster/
+
+### One or more nodes are never Ready (Network / CNI issues)
+
+You might see that one or more nodes are never ready, e.g
+
+```
```

Comment:
This, and some other fenced blocks in your patch, has a prompt, command, and output, so it should get:

    ```console

syntax highlighting (more in #27).